packer/vault-policies/write-node-policies.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+import json
+import sys
+
+NETWORK_ID = ''
```

Comment:
Do these need to be set up here?

Reply:
Not strictly up there, but I do want `NETWORK_ID`, `AWS_ACCOUNT_ID`, and `SETUP_FILE` to be global -- the values will be the same throughout the whole script execution and I'd rather not have them clutter up the function args.

Maybe I ought to move the top-level declaration down to right before the `if __name__ == '__main__'` statement?